test/server/graphql/v2/mutation/MemberInvitationMutations.test.ts

```diff
@@ -0,0 +1,413 @@
+  });
+  before(async () => {
+    collective2 = await models.Collective.create(utils.data('collective2'));
+  });
+  before(() => collective1.addUserWithRole(host, roles.HOST));
+  before(() => collective1.addUserWithRole(user1, roles.ADMIN));
+  before(() => collective2.addUserWithRole(host, roles.HOST));
+  before(() => collective2.addUserWithRole(user2, roles.ADMIN));
+
+  describe('inviteMember', () => {
+    const inviteMemberMutation = gqlV2/* GraphQL */ `
+      mutation InviteMember(
+        $memberAccount: AccountReferenceInput!
+        $account: AccountReferenceInput!
+        $role: MemberRole!
+        $description: String
+        $since: DateTime
+      ) {
+        inviteMember(
+          memberAccount: $memberAccount
+          account: $account
+          role: $role
+          description: $description
+          since: $since
+        ) {
+          id
+          role
+          description
+          since
+        }
+      }
+    `;
+    it('should create a new member invitation and its related activity', async () => {
+      sendEmailSpy.resetHistory();
+      const result = await utils.graphqlQueryV2(
+        inviteMemberMutation,
+        {
+          memberAccount: { id: idEncode(user2.id, IDENTIFIER_TYPES.ACCOUNT) },
+          account: { id: idEncode(collective1.id, IDENTIFIER_TYPES.ACCOUNT) },
+          description: 'new user 2 as MEMBER',
+          role: roles.MEMBER,
+          since: new Date('01 January 2022').toISOString(),
+        },
+        user1,
+      );
+
+      const activity = await models.Activity.findAll({
+        where: {
+          type: ActivityTypes.COLLECTIVE_CORE_MEMBER_INVITED,
+          CollectiveId: collective1.id,
+        },
+      });
+      const createdMemberInvitation = result.data.inviteMember;
+
+      expect(result.errors).to.not.exist;
+      expect(activity.length).to.equal(1);
+      expect(activity[0].data.memberCollective.id).to.equal(user2.id);
+      expect(createdMemberInvitation.role).to.equal(roles.MEMBER);
+      expect(createdMemberInvitation.description).to.equal('new user 2 as MEMBER');
+      expect(createdMemberInvitation.since.toISOString()).to.equal(new Date('01 January 2022').toISOString());
+      expect(sendEmailSpy.callCount).to.equal(1);
+      expect(sendEmailSpy.args[0][0]).to.equal(user2.email);
+      expect(sendEmailSpy.args[0][1]).to.equal("Invitation to join Scouts d'Arlon on Open Collective");
+    });
+    it('must be authenticated as an admin of the collective', async () => {
+      sendEmailSpy.resetHistory();
+      const result = await utils.graphqlQueryV2(
+        inviteMemberMutation,
+        {
+          memberAccount: { id: idEncode(user3.id, IDENTIFIER_TYPES.ACCOUNT) },
+          account: { id: idEncode(collective1.id, IDENTIFIER_TYPES.ACCOUNT) },
+          description: 'new user 2 as ADMIN',
+          role: roles.MEMBER,
+          since: new Date('01 January 2022').toISOString(),
+        },
+        user2,
```

Comment:
Here, for example, I think we could make the test clearer with a different variable name
```suggestion
        randomUser,
```